sktime/transformers/rocket.py

```diff
@@ -0,0 +1,178 @@
+from numba import njit, prange
+import numpy as np
+import pandas as pd
+
+from .base import BaseTransformer
```

Comment:
```suggestion
from sktime.base import BaseTransformer
```

Reply:
The codebase seems to use mostly absolute imports.  However, the contributing and extension guidelines both stipulate relative imports.

In any case, should this be: `from sktime.transformers.base import BaseTransformer`?

Reply:
Yes, thanks, sorry, need to update extension guidelines! We now prefer absolute imports ...